.github/workflows/InternalIssuesCreateMirror.yml

```diff
@@ -0,0 +1,82 @@
+name: Create or Label Mirror Issue
+on:
+  discussion:
+    types:
+      - labeled
+  issues:
+    types:
+      - labeled
+
+env:
+  GH_TOKEN: ${{ secrets.DUCKDBLABS_BOT_TOKEN }}
```

Comment:
Just checking: is this an org secret that's already available for duckdb-python?

Reply:
yes!